packages/gestalt-datepicker/src/LocaleDataTypes.js

```diff
@@ -0,0 +1,67 @@
+// @flow strict
+import PropTypes from 'prop-types';
+
+// flowlint unclear-type:off
+export type LocaleData = {
+  code?: string,
+  formatDistance?: (...args: Array<any>) => any,
```

Comment:
@AlbertCarreras would there be a way to be more explicit with these types? Can we avoid using `any`

Reply:
It's the official `date-fns` type.

Reply:
https://github.com/date-fns/date-fns/blob/master/src/locale/en-AU/index.js.flow